package.json

```diff
@@ -27,24 +28,19 @@
   "peerDependencies": {
     "eslint-plugin-mocha": "^2.0.0",
     "eslint-plugin-react": "^4.1.0",
-    "eslint-plugin-json": "^1.2.0",
```

Comment:
As a side note I am definitely all for maintaining backwards compatibility. Option 2 is definitely an unknown.